src/model/Car.js

```diff
@@ -0,0 +1,41 @@
+import validation from '../utils/validation.js';
+
+class Car {
+  #name;
+  #position;
+
+  constructor(name) {
+    this.#name = this.setName(name);
+    this.#position = 0;
+  }
+
+  setName(name) {
+    if (!validation.isValidateCarName(name)) return '';
```

Comment:
중괄호를 포함시켜도 괜찮을 것 같아요..! 😆
[curly](https://eslint.org/docs/latest/rules/curly)

```suggestion
    if (!validation.isValidateCarName(name)) {
      return '';
    }
```

Reply:
린트 세팅을 좀 더 꼼꼼히 해야겠습니다 ㅠㅠ!!